src/openloco/windows/construction/StationTab.cpp

```diff
@@ -0,0 +1,426 @@
+#include "Construction.h"
+
+using namespace openloco::interop;
+using namespace openloco::map;
+using namespace openloco::map::tilemgr;
+
+namespace openloco::ui::windows::construction
```

Comment:
You can just go straight to the station namespace. `namespace openloco::ui::windows::construction::station`